cpp/matrix_assembly.hpp

```diff
@@ -23,7 +23,7 @@ namespace
 /// Provides easier interface to dolfinx::fem::impl::assemble_exterior_facets
 /// @param[in] mat_set the function for setting the values in the matrix
 /// @param[in] V the function space
-/// @param[in] active_facets list of indices (local to process) of facets to be integrated over
+/// @param[in] facets list of tuples; cell index (local to process) and facet index (local to cell)
```

Comment:
It seems the function still just takes a list of active facets and the list of tuples is created inside the function.